### PR TITLE
Updating forests in regional_map_settings with new plants

### PR DESCRIPTION
**Pull request by cosmiccoincidence**
#### Summary
```SUMMARY: Content "Adds recently merged plants to regional_map_settings"```  

#### Purpose of change
- Adds new berry trees, bushes and flowers to regional_map_settings.
- Adds long grass to forest and thick forest.
- Updated long grass description.
- Added looks_like to new plants until they get sprites.

Note: This is not final as more plants including trees will be added soon and more balancing will have to be done.

---

**FOREST**
forest

**THICK FOREST**
thick forest

**SWAMP**
swamp

### CODE REVIEW

File: data/json/terrain.json [hunk 1]
```diff
@@ -523,7 +523,7 @@
     "type": "terrain",
     "id": "t_grass_long",
     "name": "long grass",
-    "description": "Long shaggy grass that hasn't been cut recently.",
+    "description": "Long shaggy grass about shin tall.",
```
Comment (Regilith) [inline data/json/terrain.json]: "shin high" is probably better used here.

**Comment by ralreegorganon** [comment]
I'm all for adding the new stuff--the one thing I wonder about is the resulting relative ratio of `t_underbrush` after the change, since I believe the forage results from them are pretty important.

I think the numbers work out like this:

#### % of `shrubs_and_flowers` that are `t_underbrush`

  terrain    | before | after
------------ | ------ | -----
forest       | 57%    | 42%
forest_thick | 67%    | 53%

If you want to keep the ratio of `t_underbrush` to everything else approximately the same, I think you can use the following weights:

  terrain    | t_underbush weight
------------ | ------------------
forest       | 19
forest_thick | 28

To be clear, I'm not opposed to there being less `t_underbrush`, I just want to make sure its a deliberate re-balancing.

**Comment by cosmiccoincidence** [comment]
Oh, hey you made your comment right as I made a commit for balancing. I appreciate your feedback so I'd be glad if you looked at it again.

**Comment by ralreegorganon** [comment]
I think the new ratios look better. I noticed that the relative ratios for the shrubs are now the same for all three terrain types, which makes them more homogeneous, but I'd rather get the new plants in and re-balance ratios in a follow up.

The additional trees and shrubs in the swamps are ok--I had originally limited the list to make them something more than just "forests with a lot more water", particularly the exclusion of the fruit trees, but I think this is fine. The real interesting thing would be the follow up, once there are different regions, to differentiate between swamps and marshlands (and in particular things like salt water marshes), at which point we can reevaluate what sorts of fauna in what ratios makes sense... but that's neither here nor there.

For the clutter in the swamp, I'd suggest maybe ditching the `f_boulder_large`--it seems a little out of place. Also, you can normalize the values, since your lowest value is a 4.

```
"t_trunk": 32,
"f_boulder_small": 32,
"f_boulder_medium": 16,
"f_boulder_large": 4
```

becomes

```
"t_trunk": 8,
"f_boulder_small": 8,
"f_boulder_medium": 4,
"f_boulder_large": 1
```

**Comment by cosmiccoincidence** [comment]
> I think the new ratios look better. I noticed that the relative ratios for the shrubs are now the same for all three terrain types, which makes them more homogeneous, but I'd rather get the new plants in and re-balance ratios in a follow up.

I agree.

> The additional trees and shrubs in the swamps are ok--I had originally limited the list to make them something more than just "forests with a lot more water", particularly the exclusion of the fruit trees,

Thinking of the fruit trees is made me decide to add the plants to swamps, I'm pretty sure fruit trees can handle swampy conditions, at least some like cherries and apples.

> For the clutter in the swamp, I'd suggest maybe ditching the f_boulder_large

Updated this.

**Comment by cosmiccoincidence** [comment]
I merged master in but it made travis build fail.

**Pull request merged**